# HG changeset patch
# User Torsten <mttl@mailbox.org>
# Date 1522872218 -7200
#      Wed Apr 04 22:03:38 2018 +0200
# Branch stable
# Node ID fc88039625333a4fb3f0756b0b4fd13b26f79ab0
# Parent  68366259ac87f7602aa5dc34f4359bd3eedf267d
updated language files with latest new strings (bug #53526)

* es_ES.ts, fr_FR.ts, nl_NL.ts

diff --git a/libgui/languages/es_ES.ts b/libgui/languages/es_ES.ts
--- a/libgui/languages/es_ES.ts
+++ b/libgui/languages/es_ES.ts
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!DOCTYPE TS>
-<TS version="2.0" language="es">
+<TS version="2.1" language="es">
 <context>
     <name>QObject</name>
     <message>
         <location filename="../src/workspace-model.cc" line="+78"/>
         <source>automatic</source>
         <translation>automático</translation>
     </message>
     <message>
@@ -50,17 +50,17 @@
         <translation>selección</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>cursor</source>
         <translation>cursor</translation>
     </message>
     <message>
-        <location filename="../src/settings-dialog.cc" line="+136"/>
+        <location filename="../src/settings-dialog.cc" line="+137"/>
         <source>Difference to the default size</source>
         <translation>Diferencia con el tamaño predeterminado</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Background color, pink (255, 0, 255) means default</source>
         <translation>Color de fondo; rosa (255,0,255) representa predeterminado</translation>
     </message>
@@ -425,31 +425,40 @@
         <location filename="../src/dialog.cc" line="+276"/>
         <source>Select All</source>
         <translation>Seleccionar todo</translation>
     </message>
 </context>
 <context>
     <name>octave::documentation</name>
     <message>
-        <location filename="../src/documentation.cc" line="+59"/>
+        <location filename="../src/documentation.cc" line="+75"/>
+        <location line="+10"/>
+        <location line="+211"/>
         <source>Octave Documentation</source>
         <translation>Documentación de Octave</translation>
     </message>
     <message>
-        <location line="+1"/>
+        <location line="-220"/>
+        <source>Could not copy help collection to temporary
+file. Search capabilities may be affected.
+%1</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+10"/>
         <source>Could not setup the data required for the
 documentation viewer. Only help texts in
 the Console Widget will be available.</source>
         <translation>No se pudieron configurar los datos necesarios 
 para el visor de documentación. Solo los textos 
 de ayuda estarán disponibles en el widget de la consola.</translation>
     </message>
     <message>
-        <location line="+20"/>
+        <location line="+19"/>
         <source>Contents</source>
         <translation>Contenido</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>Enter text to search the indices</source>
         <translation>Ingrese texto para buscar los índices</translation>
     </message>
@@ -459,16 +468,21 @@ de ayuda estarán disponibles en el widget de la consola.</translation>
         <source>Search</source>
         <translation>Búsqueda</translation>
     </message>
     <message>
         <location line="-22"/>
         <source>Index</source>
         <translation>Índice</translation>
     </message>
+    <message>
+        <location line="+157"/>
+        <source>Unable to register help file %1.</source>
+        <translation type="unfinished"></translation>
+    </message>
 </context>
 <context>
     <name>octave::documentation_dock_widget</name>
     <message>
         <location filename="../src/documentation-dock-widget.cc" line="+39"/>
         <source>Documentation</source>
         <translation>Documentación</translation>
     </message>
@@ -738,18 +752,18 @@ no puede ser abierto para escritura: %2.
     </message>
     <message>
         <location line="+5"/>
         <source>Indent Code</source>
         <translation>Aplicar sangría a código</translation>
     </message>
     <message>
         <location line="+7"/>
-        <source>convert line endings to &amp;windows (crlf)</source>
-        <translation>Convertir los finales de línea a &amp;windows (crlf)</translation>
+        <source>Convert Line Endings to &amp;Windows (CRLF)</source>
+        <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Convert Line Endings to &amp;Unix (LF)</source>
         <translation>Convertir los finales de línea a &amp;Unix (LF)</translation>
     </message>
     <message>
         <location line="+5"/>
@@ -931,21 +945,16 @@ no puede ser abierto para escritura: %2.
         <source>Recent Files</source>
         <translation>Archivos Recientes</translation>
     </message>
     <message>
         <location line="+65"/>
         <source>Editor</source>
         <translation>Editor</translation>
     </message>
-    <message>
-        <location line="+207"/>
-        <source>&amp;%1 %2</source>
-        <translation>&amp;%1 %2</translation>
-    </message>
 </context>
 <context>
     <name>octave::file_editor_tab</name>
     <message>
         <location filename="../src/m-editor/file-editor-tab.cc" line="+140"/>
         <source>line:</source>
         <translation>Línea:</translation>
     </message>
@@ -969,26 +978,26 @@ no puede ser abierto para escritura: %2.
         <location line="+231"/>
         <source>Breakpoint condition</source>
         <translation>Condición de punto de interrupción</translation>
     </message>
     <message>
         <location line="+91"/>
         <location line="+1164"/>
         <location line="+371"/>
-        <location line="+210"/>
+        <location line="+211"/>
         <location line="+20"/>
         <location line="+11"/>
         <location line="+85"/>
         <location line="+36"/>
         <source>Octave Editor</source>
         <translation>Editor de Octave</translation>
     </message>
     <message>
-        <location line="-1896"/>
+        <location line="-1897"/>
         <source>Cannot add breakpoint to modified file.
 Save and add breakpoint, or cancel?</source>
         <translation>No es posible agregar punto de interrupción al archivo modificado.
 ¿Desea guardar y agregar punto de interrupción o cancelar?</translation>
     </message>
     <message>
         <location line="+837"/>
         <source>Goto line</source>
@@ -1070,17 +1079,17 @@ Quit debugging and save?</source>
         <translation>Archivos de Octave (*.m)</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>All Files (*)</source>
         <translation>Todos los archivos (*)</translation>
     </message>
     <message>
-        <location line="+80"/>
+        <location line="+81"/>
         <source>&quot;%1&quot;
 is not a valid identifier.
 
 If you keep this filename, you will not be able to
 call your script using its name as an Octave command.
 
 Do you want to choose another name?</source>
         <translation>&quot;%1&quot;
@@ -1088,36 +1097,30 @@ no es un identificador válido.
 
 Si mantiene ese nombre de archivo no le será posible
 llamar a su script usando su nombre como comando de Octave.
 
 ¿Desea seleccionar otro nombre?</translation>
     </message>
     <message>
         <location line="+20"/>
-        <source>The current codec %1
+        <source>The current encoding %1
 can not be applied.
 
 Please select another one or cancel saving!</source>
-        <translation>El códec actual %1
-no puede ser aplicado.
-
-¡Seleccione por favor otro códec o cancele el proceso de guardado!</translation>
+        <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+11"/>
         <source>The current editor contents can not be encoded
-with the selected codec %1.
+with the selected encoding %1.
 Using it will result in data loss!
 
-Do you want to chose another codec?</source>
-        <translation>El contenido actual del editor no puede ser codificado
-con el códec seleccionado %1.
-¡Usando dicho códec provocará pérdida de datos!
-¿Desea seleccionar otro códec?</translation>
+Do you want to choose another encoding?</source>
+        <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+85"/>
         <source>It seems that &apos;%1&apos; has been modified by another application. Do you want to reload it?</source>
         <translation>Parece que el archivo &apos;%1&apos; ha sido modificado por otra aplicación. ¿Desea recargarlo?</translation>
     </message>
     <message>
         <location line="+29"/>
@@ -1171,17 +1174,17 @@ ha sido eliminado o renombrado. ¿Desea salvarlo ahora?%2</translation>
         <translation>Mostrar ocultos</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Alternating row colors</source>
         <translation>Alternar colores de filas</translation>
     </message>
     <message>
-        <location line="+25"/>
+        <location line="+26"/>
         <source>Enter the path or filename</source>
         <translatorcomment>or &quot;Introduzca la dirección o el nombre del archivo&quot;</translatorcomment>
         <translation>Introduzca la ruta o el nombre del archivo</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>One directory up</source>
         <translation>Directorio superior</translation>
@@ -1340,17 +1343,17 @@ ha sido eliminado o renombrado. ¿Desea salvarlo ahora?%2</translation>
         <location line="+1"/>
         <source>Create file in
 </source>
         <comment>String ends with \n!</comment>
         <translation>Crear archivo en
 </translation>
     </message>
     <message>
-        <location line="+1"/>
+        <location line="+2"/>
         <source>New File.txt</source>
         <translation>Nuevo archivo.txt</translation>
     </message>
     <message>
         <location line="+15"/>
         <source>Create Directory</source>
         <translation>Crear directorio</translation>
     </message>
@@ -1358,17 +1361,17 @@ ha sido eliminado o renombrado. ¿Desea salvarlo ahora?%2</translation>
         <location line="+1"/>
         <source>Create folder in
 </source>
         <comment>String ends with \n!</comment>
         <translation>Crear carpeta en
 </translation>
     </message>
     <message>
-        <location line="+1"/>
+        <location line="+2"/>
         <source>New Directory</source>
         <translation>Nuevo directorio</translation>
     </message>
 </context>
 <context>
     <name>octave::final_page</name>
     <message>
         <location filename="../src/welcome-wizard.cc" line="+274"/>
@@ -1831,38 +1834,38 @@ Pulse &apos;Siguiente&apos; para crear un archivo de configuración e iniciar Octave.&lt;/p&gt;
     </message>
     <message>
         <location line="+79"/>
         <source>Octave Community News</source>
         <translation>Noticas de la comunidad de Octave</translation>
     </message>
     <message>
         <location line="+91"/>
-        <location line="+1724"/>
+        <location line="+1747"/>
         <source>About Octave</source>
         <translation>Acerca de Octave</translation>
     </message>
     <message>
-        <location line="-1603"/>
+        <location line="-1626"/>
         <source>Octave</source>
         <translation>Octave</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Are you sure you want to exit Octave?</source>
         <translation>¿Está seguro de que desea salir de Octave?</translation>
     </message>
     <message>
         <location line="+64"/>
-        <location line="+1600"/>
+        <location line="+1623"/>
         <source>Browse directories</source>
         <translation>Explorar directorios</translation>
     </message>
     <message>
-        <location line="-1444"/>
+        <location line="-1467"/>
         <source>Octave Files (*.m);;All Files (*)</source>
         <translation>Archivos de Octave (*.m);;Todos los archivos (*)</translation>
     </message>
     <message>
         <location line="+16"/>
         <source>File Encoding:</source>
         <translation>Condificación de archivo:</translation>
     </message>
@@ -1888,17 +1891,17 @@ Pulse &apos;Siguiente&apos; para crear un archivo de configuración e iniciar Octave.&lt;/p&gt;
         <translation>No se puede encontrar la función %1</translation>
     </message>
     <message>
         <location line="+9"/>
         <source>Octave Editor</source>
         <translation>Editor de Octave</translation>
     </message>
     <message>
-        <location line="+909"/>
+        <location line="+932"/>
         <source>&amp;File</source>
         <translation>&amp;Archivo</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Open...</source>
         <translation>Abrir...</translation>
     </message>
@@ -2255,17 +2258,17 @@ o actívela para ver las novedades en el diálogo de configuración de red de Octave
         <translation>Utilice&lt;Alt&gt; + &lt;Botón izquierdo del ratón&gt; para mover la ventana</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Dock widget</source>
         <translation>Acoplar widget</translation>
     </message>
     <message>
-        <location line="+42"/>
+        <location line="+46"/>
         <source>Undock widget</source>
         <translation>Desacoplar widget</translation>
     </message>
 </context>
 <context>
     <name>octave::octave_qscintilla</name>
     <message>
         <location filename="../src/m-editor/octave-qscintilla.cc" line="+277"/>
@@ -3170,27 +3173,27 @@ Would you like to save the current short
         <location line="+3"/>
         <source>Redock</source>
         <translation>Reacoplar</translation>
     </message>
 </context>
 <context>
     <name>octave::variable_editor</name>
     <message>
-        <location line="+790"/>
+        <location line="+791"/>
         <source>Variable Editor</source>
         <translation>Editor de variables</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Edit variables.</source>
         <translation>Editar variables.</translation>
     </message>
     <message>
-        <location line="+470"/>
+        <location line="+476"/>
         <source>Variable Editor Toolbar</source>
         <translation>Barra de herramientas de editor de variables</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Save</source>
         <translation>Guardar</translation>
     </message>
@@ -3265,30 +3268,25 @@ Would you like to save the current short
         <location line="+1"/>
         <source>Go one level up in variable hierarchy</source>
         <translation>Ir un nivel hacia arriba en jerarquia de variable</translation>
     </message>
 </context>
 <context>
     <name>octave::variable_editor_stack</name>
     <message>
-        <location line="-1169"/>
+        <location line="-1176"/>
         <source>Save Variable %1 As</source>
         <translation>Guardar variable %1 como</translation>
     </message>
-    <message>
-        <location line="+2"/>
-        <source>./%1.txt</source>
-        <translation>./%1.txt</translation>
-    </message>
 </context>
 <context>
     <name>octave::variable_editor_view</name>
     <message>
-        <location line="+123"/>
+        <location line="+126"/>
         <source>Cut</source>
         <translation>Cortar</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Copy</source>
         <translation>Copiar</translation>
     </message>
@@ -3421,17 +3419,17 @@ Would you like to save the current short
         <translation>Introduzca texto para filtrar el espacio de trabajo</translation>
     </message>
     <message>
         <location line="+9"/>
         <source>Filter</source>
         <translation>Filtrar</translation>
     </message>
     <message>
-        <location line="+129"/>
+        <location line="+130"/>
         <source>View the variables in the active workspace.&lt;br&gt;</source>
         <translation>Ver las variables del espacio de trabajo activo.&lt;br&gt;</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Colors for variable attributes:</source>
         <translation>Colores para los atributos de variable:</translation>
     </message>
@@ -3560,17 +3558,22 @@ Would you like to save the current short
         <translation>Color</translation>
     </message>
     <message>
         <location line="+49"/>
         <source>Show EOL characters</source>
         <translation>Mostrar caracteres de fin de línea</translation>
     </message>
     <message>
-        <location line="+784"/>
+        <location line="+309"/>
+        <source>Wrap long lines at current window border</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+475"/>
         <source>Default EOL mode</source>
         <translation>Identificador de fin de línea por defecto</translation>
     </message>
     <message>
         <location line="-420"/>
         <source>Indent width</source>
         <translation>Ancho de sangría</translation>
     </message>
@@ -3632,22 +3635,22 @@ Would you like to save the current short
     <message>
         <location line="+280"/>
         <source>Terminal Colors</source>
         <translatorcomment>En es_ES, terminal suele ser masculino, aunque en el español iberoamericano suele adoptarse el femenino</translatorcomment>
         <translation>Colores del terminal</translation>
     </message>
     <message>
         <location line="-173"/>
-        <location line="+834"/>
+        <location line="+854"/>
         <source>Font</source>
         <translation>Tipo de fuente</translation>
     </message>
     <message>
-        <location line="-2026"/>
+        <location line="-2046"/>
         <source>Show line numbers</source>
         <translation>Mostrar números de línea</translation>
     </message>
     <message>
         <location line="+150"/>
         <source>Highlight current line</source>
         <translation>Resaltar línea actual</translation>
     </message>
@@ -3693,22 +3696,22 @@ Would you like to save the current short
     </message>
     <message>
         <location line="+1456"/>
         <source>Use foreground color</source>
         <translation>Usar color de primer plano</translation>
     </message>
     <message>
         <location line="+52"/>
-        <location line="+851"/>
+        <location line="+800"/>
         <source>Font size</source>
         <translation>Tamaño de fuente</translation>
     </message>
     <message>
-        <location line="-728"/>
+        <location line="-677"/>
         <source>Set focus to terminal when running a command from within another widget</source>
         <translation>Foco al terminal cuando se ejecute un comando desde otro widget</translation>
     </message>
     <message>
         <location line="+73"/>
         <source>File Browser</source>
         <translation>Explorador de archivos</translation>
     </message>
@@ -3881,22 +3884,17 @@ Would you like to save the current short
     </message>
     <message>
         <location line="+10"/>
         <source>Background</source>
         <translatorcomment>Segundo plano o fondo</translatorcomment>
         <translation>Segundo plano</translation>
     </message>
     <message>
-        <location line="+38"/>
-        <source>Wrap long lines at curent window border</source>
-        <translation>Envolver líneas largas en borde de la ventana actual</translation>
-    </message>
-    <message>
-        <location line="+15"/>
+        <location line="+53"/>
         <source>Indentation</source>
         <translation>Sangrado</translation>
     </message>
     <message>
         <location line="+127"/>
         <source>Indentation uses tabs</source>
         <translation>Sangrado con tabulador</translation>
     </message>
@@ -4062,72 +4060,62 @@ Would you like to save the current short
         <translation>Utilizar servidor &quot;proxy&quot;</translation>
     </message>
     <message>
         <location line="+90"/>
         <source>Variable Editor</source>
         <translation>Editor de variables</translation>
     </message>
     <message>
-        <location line="+33"/>
+        <location line="+53"/>
         <source>Default row height</source>
         <translation>Altura predeterminada de fila </translation>
     </message>
     <message>
         <location line="+17"/>
-        <source>20</source>
-        <translation>20</translation>
-    </message>
-    <message>
-        <location line="+7"/>
         <source>Default column width</source>
         <translation>Ancho predeterminado de columna</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>Use Terminal Font</source>
         <translation>Usar fuente del terminal</translation>
     </message>
     <message>
-        <location line="+27"/>
-        <source>2</source>
-        <translation>2</translation>
-    </message>
-    <message>
-        <location line="+16"/>
+        <location line="+26"/>
         <source>Autofit</source>
         <translation>Ajuste automático</translation>
     </message>
     <message>
-        <location line="+7"/>
+        <location line="-63"/>
         <source>Plus font height</source>
         <translation>Más altura de fuente</translation>
     </message>
     <message>
-        <location line="+17"/>
+        <location line="+77"/>
         <source>By Column</source>
         <translation>Por columna</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Uniform</source>
         <translation>Uniforme</translation>
     </message>
     <message>
-        <location line="+23"/>
+        <location line="+40"/>
         <source>Variable Editor Colors</source>
         <translation>Colores de editor de variables</translation>
     </message>
     <message>
         <location line="+12"/>
         <source>Use alternating row colors</source>
         <translation>Usar colores de filas alternos</translation>
     </message>
     <message>
-        <location line="-287"/>
+        <location line="-290"/>
         <source>HttpProxy</source>
         <translation>HttpProxy</translation>
     </message>
     <message>
         <location line="-2179"/>
         <source>Icon set for dock widgets</source>
         <translation>Iconos para widget</translation>
     </message>
@@ -4202,17 +4190,17 @@ Would you like to save the current short
         <translation>Nombre de usuario:</translation>
     </message>
     <message>
         <location line="+37"/>
         <source>Password:</source>
         <translation>Contraseña:</translation>
     </message>
     <message>
-        <location filename="../src/settings-dialog.cc" line="+125"/>
+        <location filename="../src/settings-dialog.cc" line="+127"/>
         <source>Octave Settings</source>
         <translation>Configuraciones de Octave</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Unable to save settings.  Missing settings file or unknown directory.</source>
         <translation>No se puede guardar la configuración.  Falta el archivo de configuración o el directorio es desconocido.</translation>
     </message>
diff --git a/libgui/languages/fr_FR.ts b/libgui/languages/fr_FR.ts
--- a/libgui/languages/fr_FR.ts
+++ b/libgui/languages/fr_FR.ts
@@ -49,17 +49,17 @@
         <translation>sélection</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>cursor</source>
         <translation>curseur</translation>
     </message>
     <message>
-        <location filename="../src/settings-dialog.cc" line="+136"/>
+        <location filename="../src/settings-dialog.cc" line="+137"/>
         <source>Difference to the default size</source>
         <translation>Différence avec la taille par défaut</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Background color, pink (255, 0, 255) means default</source>
         <translation>Couleur de fond, magenta [255,0,255] désigne la valeur par défaut {255, 0, 255)?}</translation>
     </message>
@@ -416,31 +416,40 @@
         <location filename="../src/dialog.cc" line="+276"/>
         <source>Select All</source>
         <translation>Tout Sélectionner</translation>
     </message>
 </context>
 <context>
     <name>octave::documentation</name>
     <message>
-        <location filename="../src/documentation.cc" line="+59"/>
+        <location filename="../src/documentation.cc" line="+75"/>
+        <location line="+10"/>
+        <location line="+211"/>
         <source>Octave Documentation</source>
         <translation>Documentation d&apos;Octave</translation>
     </message>
     <message>
-        <location line="+1"/>
+        <location line="-220"/>
+        <source>Could not copy help collection to temporary
+file. Search capabilities may be affected.
+%1</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+10"/>
         <source>Could not setup the data required for the
 documentation viewer. Only help texts in
 the Console Widget will be available.</source>
         <translation>Impossible de préparer les données nécessaires
 pour le navigateur de documentation. Seule l&apos;aide
 dans la console (commande &quot;help&quot;) sera accessible.</translation>
     </message>
     <message>
-        <location line="+20"/>
+        <location line="+19"/>
         <source>Contents</source>
         <translation>Contenu</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>Enter text to search the indices</source>
         <translation>Entrer le texte à rechercher dans la liste d&apos;index</translation>
     </message>
@@ -450,16 +459,21 @@ dans la console (commande &quot;help&quo
         <source>Search</source>
         <translation>Recherche</translation>
     </message>
     <message>
         <location line="-22"/>
         <source>Index</source>
         <translation>Index</translation>
     </message>
+    <message>
+        <location line="+157"/>
+        <source>Unable to register help file %1.</source>
+        <translation type="unfinished"></translation>
+    </message>
 </context>
 <context>
     <name>octave::documentation_dock_widget</name>
     <message>
         <location filename="../src/documentation-dock-widget.cc" line="+39"/>
         <source>Documentation</source>
         <translation>Documentation</translation>
     </message>
@@ -726,18 +740,18 @@ pour écrire : %2.</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Indent Code</source>
         <translation>Indenter</translation>
     </message>
     <message>
         <location line="+7"/>
-        <source>convert line endings to &amp;windows (crlf)</source>
-        <translation>Convertir les Fins de Lignes en &amp;Windows (CRLF)</translation>
+        <source>Convert Line Endings to &amp;Windows (CRLF)</source>
+        <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Convert Line Endings to &amp;Unix (LF)</source>
         <translation>Convertir les Fins de Lignes en &amp;UNIX (LF)</translation>
     </message>
     <message>
         <location line="+5"/>
@@ -919,21 +933,16 @@ pour écrire : %2.</translation>
         <source>Recent Files</source>
         <translation>Fichiers récents</translation>
     </message>
     <message>
         <location line="+65"/>
         <source>Editor</source>
         <translation>Éditeur</translation>
     </message>
-    <message>
-        <location line="+207"/>
-        <source>&amp;%1 %2</source>
-        <translation>&amp;%1 %2</translation>
-    </message>
 </context>
 <context>
     <name>octave::file_editor_tab</name>
     <message>
         <location filename="../src/m-editor/file-editor-tab.cc" line="+140"/>
         <source>line:</source>
         <translation>ligne:</translation>
     </message>
@@ -956,26 +965,26 @@ pour écrire : %2.</translation>
         <location line="+231"/>
         <source>Breakpoint condition</source>
         <translation>Condition de point d&apos;arrêt</translation>
     </message>
     <message>
         <location line="+91"/>
         <location line="+1164"/>
         <location line="+371"/>
-        <location line="+210"/>
+        <location line="+211"/>
         <location line="+20"/>
         <location line="+11"/>
         <location line="+85"/>
         <location line="+36"/>
         <source>Octave Editor</source>
         <translation>Éditeur d&apos;Octave</translation>
     </message>
     <message>
-        <location line="-1896"/>
+        <location line="-1897"/>
         <source>Cannot add breakpoint to modified file.
 Save and add breakpoint, or cancel?</source>
         <translation>Impossible d&apos;ajouter un point d&apos;arrêt à un fichier modifié.
 Sauvegarder et ajouter le point d&apos;arrêt ou annuler?</translation>
     </message>
     <message>
         <location line="+837"/>
         <source>Goto line</source>
@@ -1056,17 +1065,17 @@ Quitter le mode déboguage et sauver?</translation>
         <translation>Fichiers Octave (*.m)</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>All Files (*)</source>
         <translation>Tous les Fichiers (*)</translation>
     </message>
     <message>
-        <location line="+80"/>
+        <location line="+81"/>
         <source>&quot;%1&quot;
 is not a valid identifier.
 
 If you keep this filename, you will not be able to
 call your script using its name as an Octave command.
 
 Do you want to choose another name?</source>
         <translation>&quot;%1&quot;
@@ -1074,37 +1083,30 @@ n&apos;est pas un nom valide.
 
 Si vous gardez ce nom de fichier, vous ne pourrez pas exécuter
 votre script en utilisant son nom en guise de commande Octave.
 
 Voulez vous choisir un autre nom?</translation>
     </message>
     <message>
         <location line="+20"/>
-        <source>The current codec %1
+        <source>The current encoding %1
 can not be applied.
 
 Please select another one or cancel saving!</source>
-        <translation>Le codec %1
-ne peut être utilisé.
-
-Choisissez en un autre ou annulez la sauvegarde!</translation>
+        <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+11"/>
         <source>The current editor contents can not be encoded
-with the selected codec %1.
+with the selected encoding %1.
 Using it will result in data loss!
 
-Do you want to chose another codec?</source>
-        <translation>Le contenu actuel de l&apos;éditeur ne peut être encodé
-avec le codec sélectionné %1.
-Cela mènerait à des pertes de données!
-
-Voulez vous choisir un autre codec?</translation>
+Do you want to choose another encoding?</source>
+        <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+85"/>
         <source>It seems that &apos;%1&apos; has been modified by another application. Do you want to reload it?</source>
         <translation>Il semblerait que &apos;%1&apos; a été modifié par une autre application. Voulez vous le recharger ?</translation>
     </message>
     <message>
         <location line="+29"/>
@@ -1158,17 +1160,17 @@ a été supprimé ou renommé. Voulez vous l&apos;enregistrer maintenant ?%2</translation>
         <translation>Afficher les fichiers cachés</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Alternating row colors</source>
         <translation>Alterner les couleurs des lignes</translation>
     </message>
     <message>
-        <location line="+25"/>
+        <location line="+26"/>
         <source>Enter the path or filename</source>
         <translation>Entrez le chemin ou le nom du fichier</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>One directory up</source>
         <translation>Répertoire Parent</translation>
     </message>
@@ -1326,17 +1328,17 @@ a été supprimé ou renommé. Voulez vous l&apos;enregistrer maintenant ?%2</translation>
         <location line="+1"/>
         <source>Create file in
 </source>
         <comment>String ends with \n!</comment>
         <translation>Créer un fichier dans
 </translation>
     </message>
     <message>
-        <location line="+1"/>
+        <location line="+2"/>
         <source>New File.txt</source>
         <translation>Nouveau Fichier.txt</translation>
     </message>
     <message>
         <location line="+15"/>
         <source>Create Directory</source>
         <translation>Créer un répertoire</translation>
     </message>
@@ -1344,17 +1346,17 @@ a été supprimé ou renommé. Voulez vous l&apos;enregistrer maintenant ?%2</translation>
         <location line="+1"/>
         <source>Create folder in
 </source>
         <comment>String ends with \n!</comment>
         <translation>Créer un répertoire dans
 </translation>
     </message>
     <message>
-        <location line="+1"/>
+        <location line="+2"/>
         <source>New Directory</source>
         <translation>Nouveau répertoire</translation>
     </message>
 </context>
 <context>
     <name>octave::final_page</name>
     <message>
         <location filename="../src/welcome-wizard.cc" line="+274"/>
@@ -1812,38 +1814,38 @@ Cliquez &quot;Suivant&quot; pour créer un fichier de configuration et lancer Octave.&lt;/p&gt;
     </message>
     <message>
         <location line="+79"/>
         <source>Octave Community News</source>
         <translation>Nouvelles de la communauté Octave</translation>
     </message>
     <message>
         <location line="+91"/>
-        <location line="+1724"/>
+        <location line="+1747"/>
         <source>About Octave</source>
         <translation>À propos d&apos;Octave</translation>
     </message>
     <message>
-        <location line="-1603"/>
+        <location line="-1626"/>
         <source>Octave</source>
         <translation>Octave</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Are you sure you want to exit Octave?</source>
         <translation>Voulez vous vraiment quitter Octave ?</translation>
     </message>
     <message>
         <location line="+64"/>
-        <location line="+1600"/>
+        <location line="+1623"/>
         <source>Browse directories</source>
         <translation>Naviguer dans les Répertoires</translation>
     </message>
     <message>
-        <location line="-1444"/>
+        <location line="-1467"/>
         <source>Octave Files (*.m);;All Files (*)</source>
         <translation>Fichiers Octave (*.m);;Tous les fichiers (*)</translation>
     </message>
     <message>
         <location line="+16"/>
         <source>File Encoding:</source>
         <translation>Encodage :</translation>
     </message>
@@ -1869,17 +1871,17 @@ Cliquez &quot;Suivant&quot; pour créer un fichier de configuration et lancer Octave.&lt;/p&gt;
         <translation>Impossible de trouver la fonction %1</translation>
     </message>
     <message>
         <location line="+9"/>
         <source>Octave Editor</source>
         <translation>Éditeur d&apos;Octave</translation>
     </message>
     <message>
-        <location line="+909"/>
+        <location line="+932"/>
         <source>&amp;File</source>
         <translation>&amp;Fichier</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Open...</source>
         <translation>Ouvrir...</translation>
     </message>
@@ -2232,17 +2234,17 @@ ou activez la connexion aux nouvelles dans l&apos;onglet &quot;Réseau&quot; du dialogue de préférences d&apos;Octave.
         <translation>Utiliser &lt;Alt&gt; + &lt;Click Droit&gt; pour déplacer la fenêtre</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Dock widget</source>
         <translation>Attacher la fenêtre</translation>
     </message>
     <message>
-        <location line="+42"/>
+        <location line="+46"/>
         <source>Undock widget</source>
         <translation>Détacher la fenêtre</translation>
     </message>
 </context>
 <context>
     <name>octave::octave_qscintilla</name>
     <message>
         <location filename="../src/m-editor/octave-qscintilla.cc" line="+277"/>
@@ -3140,27 +3142,27 @@ Voulez sauvegarder les raccourcis actuel
         <location line="+3"/>
         <source>Redock</source>
         <translation>Rattacher</translation>
     </message>
 </context>
 <context>
     <name>octave::variable_editor</name>
     <message>
-        <location line="+790"/>
+        <location line="+791"/>
         <source>Variable Editor</source>
         <translation>Éditeur de Variables</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Edit variables.</source>
         <translation>Éditer les Variables.</translation>
     </message>
     <message>
-        <location line="+470"/>
+        <location line="+476"/>
         <source>Variable Editor Toolbar</source>
         <translation>Barre d&apos;outils de l&apos;éditeur de variables</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Save</source>
         <translation>Enregistrer</translation>
     </message>
@@ -3236,30 +3238,25 @@ Voulez sauvegarder les raccourcis actuel
         <location line="+1"/>
         <source>Go one level up in variable hierarchy</source>
         <translation>Monter d&apos;un niveau dans la hiérarchie de la variable</translation>
     </message>
 </context>
 <context>
     <name>octave::variable_editor_stack</name>
     <message>
-        <location line="-1169"/>
+        <location line="-1176"/>
         <source>Save Variable %1 As</source>
         <translation>Enregistrer la Variable %1 Sous</translation>
     </message>
-    <message>
-        <location line="+2"/>
-        <source>./%1.txt</source>
-        <translation>./%1.txt</translation>
-    </message>
 </context>
 <context>
     <name>octave::variable_editor_view</name>
     <message>
-        <location line="+123"/>
+        <location line="+126"/>
         <source>Cut</source>
         <translation>Couper</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Copy</source>
         <translation>Copier</translation>
     </message>
@@ -3392,17 +3389,17 @@ Voulez sauvegarder les raccourcis actuel
         <translation>Entrer du texte pour filtrer l&apos;espace de travail</translation>
     </message>
     <message>
         <location line="+9"/>
         <source>Filter</source>
         <translation>Filtrer</translation>
     </message>
     <message>
-        <location line="+129"/>
+        <location line="+130"/>
         <source>View the variables in the active workspace.&lt;br&gt;</source>
         <translation>Voir les variables dans l&apos;espace de travail.&lt;br&gt;</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Colors for variable attributes:</source>
         <translation>Couleur pour les attributs des variables:</translation>
     </message>
@@ -3526,17 +3523,22 @@ Voulez sauvegarder les raccourcis actuel
         <translation>Couleur</translation>
     </message>
     <message>
         <location line="+49"/>
         <source>Show EOL characters</source>
         <translation>Montrer les caractères de fin de ligne (EOL)</translation>
     </message>
     <message>
-        <location line="+784"/>
+        <location line="+309"/>
+        <source>Wrap long lines at current window border</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+475"/>
         <source>Default EOL mode</source>
         <translation>Fin de ligne par défaut (EOL)</translation>
     </message>
     <message>
         <location line="-420"/>
         <source>Indent width</source>
         <translation>Largeur de l&apos;indentation</translation>
     </message>
@@ -3597,22 +3599,22 @@ Voulez sauvegarder les raccourcis actuel
     </message>
     <message>
         <location line="+280"/>
         <source>Terminal Colors</source>
         <translation>Couleurs de la fenêtre de commandes</translation>
     </message>
     <message>
         <location line="-173"/>
-        <location line="+834"/>
+        <location line="+854"/>
         <source>Font</source>
         <translation>Police</translation>
     </message>
     <message>
-        <location line="-2026"/>
+        <location line="-2046"/>
         <source>Show line numbers</source>
         <translation>Afficher les numéros des lignes</translation>
     </message>
     <message>
         <location line="+150"/>
         <source>Highlight current line</source>
         <translation>Surligner la ligne courante</translation>
     </message>
@@ -3658,22 +3660,22 @@ Voulez sauvegarder les raccourcis actuel
     </message>
     <message>
         <location line="+1456"/>
         <source>Use foreground color</source>
         <translation>Utiliser la couleur de l&apos;avant plan</translation>
     </message>
     <message>
         <location line="+52"/>
-        <location line="+851"/>
+        <location line="+800"/>
         <source>Font size</source>
         <translation>Taille de police</translation>
     </message>
     <message>
-        <location line="-728"/>
+        <location line="-677"/>
         <source>Set focus to terminal when running a command from within another widget</source>
         <translation>Donner le focus au terminal en cas d&apos;exécution d&apos;une commande depuis une autre fenêtre</translation>
     </message>
     <message>
         <location line="+73"/>
         <source>File Browser</source>
         <translation>Explorateur de fichiers</translation>
     </message>
@@ -3848,22 +3850,17 @@ Voulez sauvegarder les raccourcis actuel
         <translation>Ligne</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>Background</source>
         <translation>Arrière plan</translation>
     </message>
     <message>
-        <location line="+38"/>
-        <source>Wrap long lines at curent window border</source>
-        <translation>Replier les lignes en bord de fenêtre</translation>
-    </message>
-    <message>
-        <location line="+15"/>
+        <location line="+53"/>
         <source>Indentation</source>
         <translation>Indentation</translation>
     </message>
     <message>
         <location line="+127"/>
         <source>Indentation uses tabs</source>
         <translation>Indentation par tabulation</translation>
     </message>
@@ -4029,72 +4026,62 @@ Voulez sauvegarder les raccourcis actuel
         <translation>Utiliser un serveur proxy</translation>
     </message>
     <message>
         <location line="+90"/>
         <source>Variable Editor</source>
         <translation>Éditeur de Variables</translation>
     </message>
     <message>
-        <location line="+33"/>
+        <location line="+53"/>
         <source>Default row height</source>
         <translation>Hauteur de ligne par défaut</translation>
     </message>
     <message>
         <location line="+17"/>
-        <source>20</source>
-        <translation>20</translation>
-    </message>
-    <message>
-        <location line="+7"/>
         <source>Default column width</source>
         <translation>Largeur de colonne par défaut</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>Use Terminal Font</source>
         <translation>Utiliser la police de la fenêtre de commande</translation>
     </message>
     <message>
-        <location line="+27"/>
-        <source>2</source>
-        <translation>2</translation>
-    </message>
-    <message>
-        <location line="+16"/>
+        <location line="+26"/>
         <source>Autofit</source>
         <translation>Ajustement automatique</translation>
     </message>
     <message>
-        <location line="+7"/>
+        <location line="-63"/>
         <source>Plus font height</source>
         <translation>En plus de la hauteur de police</translation>
     </message>
     <message>
-        <location line="+17"/>
+        <location line="+77"/>
         <source>By Column</source>
         <translation>Par colonne</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Uniform</source>
         <translation>Uniforme</translation>
     </message>
     <message>
-        <location line="+23"/>
+        <location line="+40"/>
         <source>Variable Editor Colors</source>
         <translation>Couleurs</translation>
     </message>
     <message>
         <location line="+12"/>
         <source>Use alternating row colors</source>
         <translation>Alterner les couleurs des lignes</translation>
     </message>
     <message>
-        <location line="-287"/>
+        <location line="-290"/>
         <source>HttpProxy</source>
         <translation>Proxy HTTP</translation>
     </message>
     <message>
         <location line="-2179"/>
         <source>Icon set for dock widgets</source>
         <translation>Jeu d&apos;icônes pour les fenêtres</translation>
     </message>
@@ -4169,17 +4156,17 @@ Voulez sauvegarder les raccourcis actuel
         <translation>Nom d&apos;utilisateur :</translation>
     </message>
     <message>
         <location line="+37"/>
         <source>Password:</source>
         <translation>Mot de passe :</translation>
     </message>
     <message>
-        <location filename="../src/settings-dialog.cc" line="+125"/>
+        <location filename="../src/settings-dialog.cc" line="+127"/>
         <source>Octave Settings</source>
         <translation>Préférences Octave</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Unable to save settings.  Missing settings file or unknown directory.</source>
         <translation>Impossible de sauvegarder les préférences.  Fichier manquant ou dossier inconnu.</translation>
     </message>
diff --git a/libgui/languages/nl_NL.ts b/libgui/languages/nl_NL.ts
--- a/libgui/languages/nl_NL.ts
+++ b/libgui/languages/nl_NL.ts
@@ -577,17 +577,17 @@ Wil je die toch verwijderen?</translatio
         <translation>selectie</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>cursor</source>
         <translation>aanwijzer</translation>
     </message>
     <message>
-        <location filename="../src/settings-dialog.cc" line="+136"/>
+        <location filename="../src/settings-dialog.cc" line="+137"/>
         <source>Difference to the default size</source>
         <translation>Verschil met standaardgrootte</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Background color, pink (255, 0, 255) means default</source>
         <translation>Achtergrondkleur; roze (255,0,255) staat voor standaardkleur {255, 0, 255)?}</translation>
     </message>
@@ -2071,31 +2071,40 @@ Wil je die overschrijven?</translation>
         <location filename="../src/dialog.cc" line="+276"/>
         <source>Select All</source>
         <translation>Selecteer alles</translation>
     </message>
 </context>
 <context>
     <name>octave::documentation</name>
     <message>
-        <location filename="../src/documentation.cc" line="+59"/>
+        <location filename="../src/documentation.cc" line="+75"/>
+        <location line="+10"/>
+        <location line="+211"/>
         <source>Octave Documentation</source>
         <translation>Octave documentatie</translation>
     </message>
     <message>
-        <location line="+1"/>
+        <location line="-220"/>
+        <source>Could not copy help collection to temporary
+file. Search capabilities may be affected.
+%1</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+10"/>
         <source>Could not setup the data required for the
 documentation viewer. Only help texts in
 the Console Widget will be available.</source>
         <translation>Kon de gegevens voor de documentatieviewer
 niet vinden. Alleen de helpteksten in de
 console widget zijn beschikbaar.</translation>
     </message>
     <message>
-        <location line="+20"/>
+        <location line="+19"/>
         <source>Contents</source>
         <translation>Inhoud</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>Enter text to search the indices</source>
         <translation>Geef tekst om in indices te zoeken</translation>
     </message>
@@ -2105,16 +2114,21 @@ console widget zijn beschikbaar.</transl
         <source>Search</source>
         <translation>Zoek</translation>
     </message>
     <message>
         <location line="-22"/>
         <source>Index</source>
         <translation>Index</translation>
     </message>
+    <message>
+        <location line="+157"/>
+        <source>Unable to register help file %1.</source>
+        <translation type="unfinished"></translation>
+    </message>
 </context>
 <context>
     <name>octave::documentation_dock_widget</name>
     <message>
         <location filename="../src/documentation-dock-widget.cc" line="+39"/>
         <source>Documentation</source>
         <translation>Documentatie</translation>
     </message>
@@ -2381,18 +2395,18 @@ niet openen om te schrijven: %2.</transl
     </message>
     <message>
         <location line="+5"/>
         <source>Indent Code</source>
         <translation>Code laten inspringen</translation>
     </message>
     <message>
         <location line="+7"/>
-        <source>convert line endings to &amp;windows (crlf)</source>
-        <translation>Zet regeleindes om naar Windows (CRLF)</translation>
+        <source>Convert Line Endings to &amp;Windows (CRLF)</source>
+        <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Convert Line Endings to &amp;Unix (LF)</source>
         <translation>Zet regeleindes om naar &amp;Unix (LF)</translation>
     </message>
     <message>
         <location line="+5"/>
@@ -2574,21 +2588,16 @@ niet openen om te schrijven: %2.</transl
         <source>Recent Files</source>
         <translation>Recente bestanden</translation>
     </message>
     <message>
         <location line="+65"/>
         <source>Editor</source>
         <translation>Editor</translation>
     </message>
-    <message>
-        <location line="+207"/>
-        <source>&amp;%1 %2</source>
-        <translation>&amp;%1 %2</translation>
-    </message>
 </context>
 <context>
     <name>octave::file_editor_tab</name>
     <message>
         <location filename="../src/m-editor/file-editor-tab.cc" line="+140"/>
         <source>line:</source>
         <translation>regel:</translation>
     </message>
@@ -2611,26 +2620,26 @@ niet openen om te schrijven: %2.</transl
         <location line="+231"/>
         <source>Breakpoint condition</source>
         <translation>Onderbrekingspunt voorwaarde</translation>
     </message>
     <message>
         <location line="+91"/>
         <location line="+1164"/>
         <location line="+371"/>
-        <location line="+210"/>
+        <location line="+211"/>
         <location line="+20"/>
         <location line="+11"/>
         <location line="+85"/>
         <location line="+36"/>
         <source>Octave Editor</source>
         <translation>Octave Editor</translation>
     </message>
     <message>
-        <location line="-1896"/>
+        <location line="-1897"/>
         <source>Cannot add breakpoint to modified file.
 Save and add breakpoint, or cancel?</source>
         <translation>Kan geen onderbrekingpunt in een aangepast bestand plaatsen.
 Opslaan en onderbrekingspunt plaatsen, of annuleren?</translation>
     </message>
     <message>
         <location line="+837"/>
         <source>Goto line</source>
@@ -2711,17 +2720,17 @@ Debuggen afsluiten en opslaan?</translat
         <translation>Octave bestanden (*.m)</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>All Files (*)</source>
         <translation>Alle bestanden (*)</translation>
     </message>
     <message>
-        <location line="+80"/>
+        <location line="+81"/>
         <source>&quot;%1&quot;
 is not a valid identifier.
 
 If you keep this filename, you will not be able to
 call your script using its name as an Octave command.
 
 Do you want to choose another name?</source>
         <translation>&quot;%1&quot;
@@ -2729,37 +2738,30 @@ is geen geldige identifier.
 
 Als je deze bestandsnaam houdt, kan je dit script 
 niet aanroepen met de naam als Octave commando.
 
 Wil je een andere naam kiezen?</translation>
     </message>
     <message>
         <location line="+20"/>
-        <source>The current codec %1
+        <source>The current encoding %1
 can not be applied.
 
 Please select another one or cancel saving!</source>
-        <translation>The huidige codec %1
-kan niet worden toegepast.
-
-Selecteer s.v.p. een andere of annuleer opslaan!</translation>
+        <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+11"/>
         <source>The current editor contents can not be encoded
-with the selected codec %1.
+with the selected encoding %1.
 Using it will result in data loss!
 
-Do you want to chose another codec?</source>
-        <translation>De huidige gegevens in de editor kunnen niet 
-gecodeerd worden met de geselecteerde codec %1.
-Deze gebruiken resulteert in verlies van gegevens!
-
-Wil je een andere codec kiezen?</translation>
+Do you want to choose another encoding?</source>
+        <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+85"/>
         <source>It seems that &apos;%1&apos; has been modified by another application. Do you want to reload it?</source>
         <translation>Het lijkt erop dat &apos;%1&apos; is gewijzigd door een ander programma. Wil je het opnieuw laden?</translation>
     </message>
     <message>
         <location line="+29"/>
@@ -2813,17 +2815,17 @@ is gewist of hernoemd. Wil je het nu ops
         <translation>Toon verborgen bestanden</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Alternating row colors</source>
         <translation>Om en om andere regelkleuren</translation>
     </message>
     <message>
-        <location line="+25"/>
+        <location line="+26"/>
         <source>Enter the path or filename</source>
         <translation>Voer pad- of bestandsnaam in</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>One directory up</source>
         <translation>Eén mapniveau omhoog</translation>
     </message>
@@ -2981,17 +2983,17 @@ is gewist of hernoemd. Wil je het nu ops
         <location line="+1"/>
         <source>Create file in
 </source>
         <comment>String ends with \n!</comment>
         <translation>Maak nieuw bestand in
 </translation>
     </message>
     <message>
-        <location line="+1"/>
+        <location line="+2"/>
         <source>New File.txt</source>
         <translation>New file.txt</translation>
     </message>
     <message>
         <location line="+15"/>
         <source>Create Directory</source>
         <translation>Maak nieuwe map</translation>
     </message>
@@ -2999,17 +3001,17 @@ is gewist of hernoemd. Wil je het nu ops
         <location line="+1"/>
         <source>Create folder in
 </source>
         <comment>String ends with \n!</comment>
         <translation>Maak nieuwe map in
 </translation>
     </message>
     <message>
-        <location line="+1"/>
+        <location line="+2"/>
         <source>New Directory</source>
         <translation>Nieuwe map</translation>
     </message>
 </context>
 <context>
     <name>octave::final_page</name>
     <message>
         <location filename="../src/welcome-wizard.cc" line="+274"/>
@@ -3467,38 +3469,38 @@ Klik &apos;Volgende&apos; om een configu
     </message>
     <message>
         <location line="+79"/>
         <source>Octave Community News</source>
         <translation>Octave Community Nieuws</translation>
     </message>
     <message>
         <location line="+91"/>
-        <location line="+1724"/>
+        <location line="+1747"/>
         <source>About Octave</source>
         <translation>Over Octave</translation>
     </message>
     <message>
-        <location line="-1603"/>
+        <location line="-1626"/>
         <source>Octave</source>
         <translation>Octave</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Are you sure you want to exit Octave?</source>
         <translation>Zeker weten dat je Octave wil afsluiten?</translation>
     </message>
     <message>
         <location line="+64"/>
-        <location line="+1600"/>
+        <location line="+1623"/>
         <source>Browse directories</source>
         <translation>Blader door mappen</translation>
     </message>
     <message>
-        <location line="-1444"/>
+        <location line="-1467"/>
         <source>Octave Files (*.m);;All Files (*)</source>
         <translation>Octave bestanden (*.m);;Alle bestanden (*)</translation>
     </message>
     <message>
         <location line="+16"/>
         <source>File Encoding:</source>
         <translation>Bestandscodering</translation>
     </message>
@@ -3525,17 +3527,17 @@ Klik &apos;Volgende&apos; om een configu
         <translation>Kan functie %1 niet vinden</translation>
     </message>
     <message>
         <location line="+9"/>
         <source>Octave Editor</source>
         <translation>Octave Editor</translation>
     </message>
     <message>
-        <location line="+909"/>
+        <location line="+932"/>
         <source>&amp;File</source>
         <translation>Bestand</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Open...</source>
         <translation>Open...</translation>
     </message>
@@ -3888,17 +3890,17 @@ of schakel de internetverbinding voor ni
         <translation>Gebruik &lt;Alt&gt; + &lt;Linker Muisknop&gt; om het venster te verschuiven</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Dock widget</source>
         <translation>Widget in venster opnemen</translation>
     </message>
     <message>
-        <location line="+42"/>
+        <location line="+46"/>
         <source>Undock widget</source>
         <translation>Widget laten zweven</translation>
     </message>
 </context>
 <context>
     <name>octave::octave_qscintilla</name>
     <message>
         <location filename="../src/m-editor/octave-qscintilla.cc" line="+277"/>
@@ -4796,27 +4798,27 @@ Wil je de huidige sneltoetsen opslaan of
         <location line="+3"/>
         <source>Redock</source>
         <translation>Terug docken</translation>
     </message>
 </context>
 <context>
     <name>octave::variable_editor</name>
     <message>
-        <location line="+790"/>
+        <location line="+791"/>
         <source>Variable Editor</source>
         <translation>Variable Editor</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Edit variables.</source>
         <translation>Variabelen bewerken</translation>
     </message>
     <message>
-        <location line="+470"/>
+        <location line="+476"/>
         <source>Variable Editor Toolbar</source>
         <translation>Variable Editor Werkbalk</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Save</source>
         <translation>Bewaar</translation>
     </message>
@@ -4891,30 +4893,25 @@ Wil je de huidige sneltoetsen opslaan of
         <location line="+1"/>
         <source>Go one level up in variable hierarchy</source>
         <translation>Ga een niveau omhoog in variabele hierarchie</translation>
     </message>
 </context>
 <context>
     <name>octave::variable_editor_stack</name>
     <message>
-        <location line="-1169"/>
+        <location line="-1176"/>
         <source>Save Variable %1 As</source>
         <translation>Sla variable %1 op als</translation>
     </message>
-    <message>
-        <location line="+2"/>
-        <source>./%1.txt</source>
-        <translation>./%1.txt</translation>
-    </message>
 </context>
 <context>
     <name>octave::variable_editor_view</name>
     <message>
-        <location line="+123"/>
+        <location line="+126"/>
         <source>Cut</source>
         <translation>Knippen</translation>
     </message>
     <message>
         <location line="+4"/>
         <source>Copy</source>
         <translation>Kopiëren</translation>
     </message>
@@ -5047,17 +5044,17 @@ Wil je de huidige sneltoetsen opslaan of
         <translation>Geef patroon voor filteren van werkruimte</translation>
     </message>
     <message>
         <location line="+9"/>
         <source>Filter</source>
         <translation>Filter</translation>
     </message>
     <message>
-        <location line="+129"/>
+        <location line="+130"/>
         <source>View the variables in the active workspace.&lt;br&gt;</source>
         <translation>Bekijk variabelen in actieve werkruimte.&lt;br&gt;</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Colors for variable attributes:</source>
         <translation>Kleuren voor eigenschappen van variabelen:</translation>
     </message>
@@ -5181,17 +5178,22 @@ Wil je de huidige sneltoetsen opslaan of
         <translation>Kleur</translation>
     </message>
     <message>
         <location line="+49"/>
         <source>Show EOL characters</source>
         <translation>Toon EOL tekens</translation>
     </message>
     <message>
-        <location line="+784"/>
+        <location line="+309"/>
+        <source>Wrap long lines at current window border</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+475"/>
         <source>Default EOL mode</source>
         <translation>Standaard EOL instelling</translation>
     </message>
     <message>
         <location line="-420"/>
         <source>Indent width</source>
         <translation>Inspringafstand</translation>
     </message>
@@ -5252,22 +5254,22 @@ Wil je de huidige sneltoetsen opslaan of
     </message>
     <message>
         <location line="+280"/>
         <source>Terminal Colors</source>
         <translation>Terminalkleuren</translation>
     </message>
     <message>
         <location line="-173"/>
-        <location line="+834"/>
+        <location line="+854"/>
         <source>Font</source>
         <translation>Lettertype</translation>
     </message>
     <message>
-        <location line="-2026"/>
+        <location line="-2046"/>
         <source>Show line numbers</source>
         <translation>Toon regelnummers</translation>
     </message>
     <message>
         <location line="+150"/>
         <source>Highlight current line</source>
         <translation>Markeer huidige regel</translation>
     </message>
@@ -5313,22 +5315,22 @@ Wil je de huidige sneltoetsen opslaan of
     </message>
     <message>
         <location line="+1456"/>
         <source>Use foreground color</source>
         <translation>Gebruik voorgrondkleur</translation>
     </message>
     <message>
         <location line="+52"/>
-        <location line="+851"/>
+        <location line="+800"/>
         <source>Font size</source>
         <translation>Lettergrootte</translation>
     </message>
     <message>
-        <location line="-728"/>
+        <location line="-677"/>
         <source>Set focus to terminal when running a command from within another widget</source>
         <translation>Zet focus op terminal wanneer een opdracht uit een ander widget wordt uitgevoerd</translation>
     </message>
     <message>
         <location line="+73"/>
         <source>File Browser</source>
         <translation>Bestandsbrowser</translation>
     </message>
@@ -5460,22 +5462,22 @@ Wil je de huidige sneltoetsen opslaan of
     <message>
         <location line="+65"/>
         <source>Comments (Octave)</source>
         <translation>Commentaar (Octave)</translation>
     </message>
     <message>
         <location line="+11"/>
         <source>Strings considered for uncommenting text</source>
-        <translation>Strings voor "uncommenting" tekst</translation>
+        <translation>Strings voor &quot;uncommenting&quot; tekst</translation>
     </message>
     <message>
         <location line="+23"/>
         <source>String used for commenting selected text</source>
-        <translation>Strings voor "commenting" tekst</translation>
+        <translation>Strings voor &quot;commenting&quot; tekst</translation>
     </message>
     <message>
         <location line="+31"/>
         <source>Long lines</source>
         <translation>Lange regels</translation>
     </message>
     <message>
         <location line="+25"/>
@@ -5503,22 +5505,17 @@ Wil je de huidige sneltoetsen opslaan of
         <translation>Regel</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>Background</source>
         <translation>Achtergrond</translation>
     </message>
     <message>
-        <location line="+38"/>
-        <source>Wrap long lines at curent window border</source>
-        <translation>Breek lange regels af op vensterrand</translation>
-    </message>
-    <message>
-        <location line="+15"/>
+        <location line="+53"/>
         <source>Indentation</source>
         <translation>Inspringen</translation>
     </message>
     <message>
         <location line="+127"/>
         <source>Indentation uses tabs</source>
         <translation>Inspringen m.b.v. tabs</translation>
     </message>
@@ -5684,72 +5681,62 @@ Wil je de huidige sneltoetsen opslaan of
         <translation>Gebruik proxyserver</translation>
     </message>
     <message>
         <location line="+90"/>
         <source>Variable Editor</source>
         <translation>Variable Editor</translation>
     </message>
     <message>
-        <location line="+33"/>
+        <location line="+53"/>
         <source>Default row height</source>
         <translation>Standaard regelhoogte</translation>
     </message>
     <message>
         <location line="+17"/>
-        <source>20</source>
-        <translation>3D {20?}</translation>
-    </message>
-    <message>
-        <location line="+7"/>
         <source>Default column width</source>
         <translation>Standaard kolombreedte</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>Use Terminal Font</source>
         <translation>Gebruik zelfde font als Terminal</translation>
     </message>
     <message>
-        <location line="+27"/>
-        <source>2</source>
-        <translation>3D {2?}</translation>
-    </message>
-    <message>
-        <location line="+16"/>
+        <location line="+26"/>
         <source>Autofit</source>
         <translation>Automatisch aanpassen</translation>
     </message>
     <message>
-        <location line="+7"/>
+        <location line="-63"/>
         <source>Plus font height</source>
         <translation>Plus fonthoogte</translation>
     </message>
     <message>
-        <location line="+17"/>
+        <location line="+77"/>
         <source>By Column</source>
         <translation>Per kolom</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>Uniform</source>
         <translation>Uniform</translation>
     </message>
     <message>
-        <location line="+23"/>
+        <location line="+40"/>
         <source>Variable Editor Colors</source>
         <translation>Variable Editor Kleuren</translation>
     </message>
     <message>
         <location line="+12"/>
         <source>Use alternating row colors</source>
         <translation>Gebruik alternerende rijkleuren</translation>
     </message>
     <message>
-        <location line="-287"/>
+        <location line="-290"/>
         <source>HttpProxy</source>
         <translation>http proxy</translation>
     </message>
     <message>
         <location line="-2179"/>
         <source>Icon set for dock widgets</source>
         <translation>Iconenset voor dock widgets</translation>
     </message>
@@ -5824,17 +5811,17 @@ Wil je de huidige sneltoetsen opslaan of
         <translation>Gebruikersnaam:</translation>
     </message>
     <message>
         <location line="+37"/>
         <source>Password:</source>
         <translation>Wachtwoord:</translation>
     </message>
     <message>
-        <location filename="../src/settings-dialog.cc" line="+125"/>
+        <location filename="../src/settings-dialog.cc" line="+127"/>
         <source>Octave Settings</source>
         <translation>Octave Instellingen</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>Unable to save settings.  Missing settings file or unknown directory.</source>
         <translation>Kan instellingen niet opslaan.  Ontbrekend instellingenbestand of onbekende map.</translation>
     </message>
